packages/loot-core/src/server/aql/compiler.ts

```diff
@@ -312,6 +321,23 @@ function castInput(state, expr, type) {
         'date-year',
       );
     }
+  } else if (type === 'date-week') {
```

Comment:
Should we drop this as well if it's not being used?

Reply:
Yep, shoulda noticed that. Thanks!